mx.sulong/mx_sulong.py

```diff
@@ -363,6 +363,11 @@ def runLLVM(args=None, out=None):
     vmArgs, sulongArgs = truffle_extract_VM_args(args)
     return mx.run_java(getCommonOptions(False) + vmArgs + getClasspathOptions() + ["com.oracle.truffle.llvm.Sulong"] + sulongArgs, out=out)
 
+def injectBitcode(args=None, out=None):
+    """Injects the bitcode of a Mach-O WLLVM build into the executable"""
+    vmArgs, injectorArgs = truffle_extract_VM_args(args)
+    return mx.run_java(getCommonOptions(False) + vmArgs + mx.get_runtime_jvm_args('BITCODE_INJECTOR') + ["com.oracle.truffle.llvm.parser.macho.injector.Injector"] + injectorArgs, out=out)
```

Comment:
Do we actually need the `getCommonOptions` here, since the code is not really Truffle/Graal related?